lib/graph.py

```diff
@@ -66,18 +75,14 @@ def __str__(self):
                 for key, value in self.style.items()]))
 
 
-
 class CommandNode(Node):
-    def __init__(self, command, truncate=30, **style):
-        self.command = command
+    def __init__(self, description, command, truncate=40, **style):
         self.id = hash(command)
+        self.description = wrap_command(truncate-1, '<BR/>', description)
```

Comment:
Suggest to also make the description bold, so that it's better distinguished from the command?